lib/mongoose.js

```diff
@@ -35,7 +35,7 @@ module.exports = function FilterablePlugin(Filterable, schema, options) {
             return [
                 field,
                 {
-                    type: _.contains(VALID_TYPES, schemaField.type)? schemaField.type : null,
+                    type: _.contains(VALID_TYPES, schemaField.instance)? schemaField.type : null,
```

Comment:
The comment you referenced is outdated. `schemaField.type` always returns undefined. Since that comment `schemaType.instance` has been expanded to return all types. See [here](https://github.com/Automattic/mongoose/pull/2727).